Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -735,11 +738,35 @@ def get_mapping_fields_command():
     indexes = FETCH_INDEX.split(',')
     elastic_mapping = {}
     for index in indexes:
-        res = requests.get(SERVER + '/' + index + '/_mapping', auth=(USERNAME, PASSWORD), verify=INSECURE)
-        my_map = res.json()[index]['mappings']['properties']
-        elastic_mapping[index] = {"_id": "doc_id", "_index": index}
-        elastic_mapping[index]["_source"] = parse_subtree(my_map)
-    demisto.results(elastic_mapping)
+        if index == '':
+            res = requests.get(SERVER + '/_mapping', auth=(USERNAME, PASSWORD), verify=INSECURE)
```

Comment:
Please remove the handling of empty indexes. It's expected to be filled when using `fetch-incidents`.
See `fetch_params_check()`.